presets/README.md

```diff
@@ -16,3 +16,69 @@ Each preset model has its own hardware requirements in terms of GPU count and GP
 ## Distributed inference
 
 For models that support distributed inference, when the node count is larger than one, [torch distributed elastic](https://pytorch.org/docs/stable/distributed.elastic.html) is configured with master/worker pods running in multiple nodes and the service endpoint is the master pod.
+
+## OOM Troubleshooting
+
+### Gathering Debug Information
+1. Set CUDA Environment Variables:
+   - `CUDA_LAUNCH_BLOCKING=1`: This variable provides more detailed debug information
+   - Reduce GPU Memory Fragmentation by setting:
```

Comment:
Is this about debugging setup? This looks like some defaults that we should use in the pod?

Reply:
Are you recommending cx to set these? How can they set it? pros and cons of setting them?

Reply:
Updated